spg/Animal/lib/RaceHorse.pm

```diff
@@ -77,34 +83,24 @@ sub new {
 
   @{$self}{@stats} = @{$stats_data}{@stats};
 
-  $self;
-}
-
-sub won { $_[0]->{wins}++; }
-
-sub placed { $_[0]->{places}++; }
-
-sub showed { $_[0]->{shows}++; }
-
-sub lost { $_[0]->{losses}++; }
-
-sub standings {
-  my ( $self ) = @_;
-  join ', ', map "$self->{$_} $_", @stats;
+  return $self;
 }

```

Comment:
Yes, because you call `SUPER::new` it calls both the `Horse` constructor and the Moose one, which properly initialises your attributes. Using `BUILD` just saves you from having to explicitly call `SUPER::new`, and might be considered more idiomatic by Moosey people.